### ISSUE TITLE
Enrich slim variants with standard tools?

### ISSUE THREAD

**Issue by ynohat** [akamai/akamai-docker#35]
Recent implementations of these images in github actions show that we are defaulting to the full-fat akamai/shell.

I suggest that in many cases this is just because we don't have jq, bash.... Could we switch to slim images if we implemented some of these in the base image?

This has no significant impact, except that we will limit the number of bytes downloaded by GH actions, thereby reducing our indirect carbon footprint :)

**Comment by javiergarza** [comment]
Yes, this was on my mind too. I definitely support the idea of adding bash, jq and HTTPie to most of the CLI images so they can be leveraged by GitHub actions (maybe with the exception of the purge image)
